plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/transactionlog/TableSnapshot.java

```diff
@@ -260,6 +278,97 @@ private CheckpointEntryIterator getCheckpointTransactionLogEntries(
                 domainCompactionThreshold,
                 partitionConstraint,
                 addStatsMinMaxColumnFilter);
+        return stream(checkpointEntryIterator)
```

Comment:
wrapping doesn't improve readability. let's move onClose to the same line as stream()